### PR TITLE
DAT-3934 adjust europa theme colors and font sizes

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 1]
```diff
@@ -97,42 +77,46 @@ $tabber-background-color: mix($black, $color-buttons, $background-color-mix * 4)
...
-		text-align: left;
+		li {
+			line-height: $list-item-line-height;
+			list-style-type: disc;
+			padding-bottom: $list-item-spacing;
+		}
 
-		.current {
-			background-color: $color-buttons;
+		li:last-child {
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss]: `&:last-child`

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 2]
```diff
@@ -97,42 +77,46 @@ $tabber-background-color: mix($black, $color-buttons, $background-color-mix * 4)
...
+	.pi-tab-link {
+		padding: $infobox-header-spacing $infobox-side-spacing;
+	}
+
+	.pi-header,
+	.pi-title {
+		background-color: $color-buttons;
+		color: $color-button-text;
+		text-align: center;
+		a {
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss:131]: what about a new line before `a`?

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 3]
```diff
@@ -152,15 +136,52 @@ $tabber-background-color: mix($black, $color-buttons, $background-color-mix * 4)
 		}
 	}
 
+	.pi-horizontal-group-item {
+		&:nth-child(even) {
+			background-color: $horizontal-group-stripe-background;
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss]: we can merge this selector to avoid nesting

**Comment by RafLeszczynski** [comment]
code changes - LGTM
what about broken tests ?

**Comment by sebastianmarzjan** [comment]
LGTM

**Comment by sebastianmarzjan** [comment]
LG

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 4]
```diff
@@ -80,13 +80,25 @@ $tabber-background-color: mix($color-buttons, $color-page, $background-color-mix
...
-			list-style-type: disc;
 			padding-bottom: $list-item-spacing;
 
 			&:last-child {
 				padding-bottom: 0;
 			}
 		}
+
+		ul {
+			list-style-type: disc;
```
Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss:93]: Is that explicitly needed for ul?

**Comment by sebastianmarzjan** [comment]
One minor remark, LGTM

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 5]
```diff
@@ -13,23 +13,26 @@ $header-font-size: 16px;
...
+$tab-font-size: 14px;
+$title-font-size: 20px;
+$title-line-height: 24px;
 $white: #fff;
 
 // color mixes
-$background-color: mix($black, $color-page, $background-color-mix);
-$horizontal-group-stripe-background: mix($black, $color-page, $background-color-mix * 2);
-$tabber-background-color: mix($black, $color-buttons, $background-color-mix * 4);
+// 5% color, 5% black/white, 90% background
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss]: `5% button color`

**Comment by dianafa** [comment]
besides this 2 nitpicks LGTM :ship:

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 6]
```diff
@@ -29,7 +29,7 @@ $tabber-background-color: mix($color-buttons, $color-page, $background-color-mix
 @if $is-dark-wiki {
 	$background-color: mix(mix($color-buttons, $white, $background-color-mix * 5), $color-page, $background-color-mix);
 }
-$horizontal-group-stripe-background: mix($color-buttons, $background-color, $background-color-mix / 2);
+$horizontal-group-stripe-background: mix($color-buttons, $background-color, $background-color-mix);
```
Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss:32]: This is now 10% color-buttons and 90% background-color. Is this intentional?

**Comment by sebastianmarzjan** [comment]
Ok, seems legit

File: extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss [hunk 7]
```diff
@@ -13,23 +13,28 @@ $header-font-size: 16px;
 $header-line-height: 20px;
 $horizontal-group-label-spacing: 6px;
 $label-width: 88px;
+$list-item-line-height: 18px;
+$list-item-spacing: 8px;
 $tabber-lightness-condition: 30;
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss]: do we still need this var?

Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss]: no :)

**Comment by dianafa** [comment]
LGTM

**Pull request merged**